### ISSUE TITLE
Better explanation of fixpoint errors

### ISSUE THREAD

**Issue by ibdknox** [witheve/Eve#648]
We need a better explanation of fixpoint errors when they happen. Assuming we can continue to keep the changes at each logical timestep as we do now, we should be able to very nearly explain in plain english exactly what's going on.

**Comment by shamrin** [comment]
(This issue was created in response to [Puzzles & Paradoxes IV: Overbound](https://groups.google.com/d/msg/eve-talk/7lhPSjHU3kM/XqqOUJnhCQAJ).)
